docs/source/plot.rst

```diff
@@ -78,18 +96,23 @@ Assuming the file **areas.yaml** has the following area definition:
 **Example usage:**
 
 .. doctest::
+   :hide:
+ >>> from pyresample.geometry import AreaDefinition
+ >>> area_id = 'pc_world'
+ >>> description = 'Plate Carree world map'
+ >>> proj_id = 'eqc'
+ >>> projection = {'proj': 'eqc', 'lat_0': -40, 'lon_0': 40, 'a': 6370997.0, 'units': 'm'}
+ >>> width = 640
+ >>> height = 480
+ >>> area_extent = (-20037508.34, -10018754.17, 20037508.34, 10018754.17)
+ >>> area_def = AreaDefinition(area_id, description, proj_id, projection,
+ ...                           width, height, area_extent)
 
- >>> import numpy as np 
- >>> from pyresample import load_area, save_quicklook, SwathDefinition
- >>> from pyresample.kd_tree import resample_nearest
- >>> lons = np.zeros(1000)
- >>> lats = np.arange(-80, -90, -0.01)
- >>> tb37v = np.arange(1000)
- >>> area_def = load_area('areas.yaml', 'pc_world')
- >>> swath_def = SwathDefinition(lons, lats)
+ >>> area_def = load_area('areas.yaml', 'pc_world') # doctest: +SKIP
```

Comment:
Isn't this area load just overwriting the area_def created in the lines above?

Reply:
Yes, right. But the stuff above is hidden, and does not show up on RTD. Therefore the additional line for RTD. The area is not defined and on Travis we don't have access to `areas.yaml` and certainly not in the home dir.

Reply:
Ah, pyresample doesn't have a builtin areas :man_facepalming: Got it.